cmd/flarectl/dns.go

```diff
@@ -83,7 +83,7 @@ func dnsCreateOrUpdate(c *cli.Context) error {
 		return err
 	}
 
-	records, _, err := api.ListDNSRecords(context.Background(), cloudflare.ZoneIdentifier(zoneID), cloudflare.ListDNSRecordsParams{Name: name + "." + zone})
+	records, _, err := api.ListDNSRecords(context.Background(), cloudflare.ZoneIdentifier(zoneID), cloudflare.ListDNSRecordsParams{Name: name})
```

Comment:
On line 117, the record itself is create with just `Name: name`, not zone appended to it